components/raftstore-v2/src/raft/peer.rs

```diff
@@ -100,6 +100,11 @@ pub struct Peer<EK: KvEngine, ER: RaftEngine> {
     // Trace which peers have not finished split.
     split_trace: Vec<(u64, HashSet<u64>)>,
     split_flow_control: SplitFlowControl,
+    /// `MsgRequestPreVote`, `MsgRequestVote` or `MsgAppend` messages from newly
```

Comment:
minor: comments should not include prevote and vote. 
And split_pending_msg can be renamed as split_pending_append_msg.